dape.el

```diff
@@ -199,7 +199,50 @@
      :request "launch"
      :cwd dape-cwd-fn
      :program dape-find-file
-     :stopAtEntry nil))
+     :stopAtEntry nil)
+    (jdtls
```

Comment:
I added function in https://github.com/svaante/dape/pull/34/commits/a42db38ae7c27b05f4d89bec253f4e8ea1fb0ac6. It starts eglot if it is not found running for the target buffer.

There is still an issue with re-running Dape ("d" command) from the REPL: because the REPL buffer is no Java buffer, there will not be an eglot server there. I don't see how this could be fixed at the moment.